manifests/k/kramo/Cartridges/v1.3.2/kramo.Cartridges.installer.yaml

```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: kramo.Cartridges
+PackageVersion: v1.3.2
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://github.com/kra-mo/cartridges/releases/download/v1.3.2/Cartridges.Setup.exe
+  InstallerSha256: 5D7FED7EE8BA32D7661E1602EE0F0EBD551A0911C25EA9987A8154CF57F5C689

```

Comment:
```suggestion
InstallerType: inno
Installers:
- Architecture: x86
  Scope: user
  InstallerUrl: https://github.com/kra-mo/cartridges/releases/download/v1.3.2/Cartridges.Setup.exe
  InstallerSha256: 5D7FED7EE8BA32D7661E1602EE0F0EBD551A0911C25EA9987A8154CF57F5C689
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x86
  Scope: machine
  InstallerUrl: https://github.com/kra-mo/cartridges/releases/download/v1.3.2/Cartridges.Setup.exe
  InstallerSha256: 5D7FED7EE8BA32D7661E1602EE0F0EBD551A0911C25EA9987A8154CF57F5C689
  InstallerSwitches:
    Custom: /ALLUSERS
```

Reply:
Currently the package prompts for scope: 
![InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230407185551](https://user-images.githubusercontent.com/114952565/230665295-0af2a09f-416e-47bd-a6b3-d9251a5d199f.png)
``

Reply:
Thanks for the help, I will apply the changes.